SafeTrade/plugins/users/start.py

```diff
@@ -50,6 +59,16 @@ async def start(_, message: Message):
     )
 
 
+@Client.on_message(filters.command(["about_us"]))
```

Comment:
we dont need the command, remove it 
Button is enough

Reply:
that was one of the things i wanted to ask
you got it